### ISSUE TITLE
Missing protection diode

### ISSUE THREAD

**Issue by jordens** [sinara-hw/Laser_Module#7]
There should also be a shottky diode antiparallel to the laser diode.

This is different from the forward diode in #6

**Comment by gkasprow** [comment]
you mean such connection?

![obraz](https://user-images.githubusercontent.com/4325054/235147854-e66fc97f-e78d-4a5b-932e-3cac312128fd.png)

**Comment by jordens** [comment]
@nkrackow @j-hirsch

**Comment by nkrackow** [comment]
Yes this looks good. Just the inductors are still inline towards the laser diode now.
